1.0/profile.ttl

```diff
@@ -0,0 +1,63 @@
+PREFIX dcterms: <http://purl.org/dc/terms/>
+PREFIX owl: <http://www.w3.org/2002/07/owl#>
+PREFIX prof: <http://www.w3.org/ns/dx/prof/>
+PREFIX role: <http://www.w3.org/ns/dx/prof/role/>
```

Comment:
URIs using this prefix, e.g. http://www.w3.org/ns/dx/prof/role/scheme, do not resolve.  This makes it hard to check if the assigned roles make sense.

Reply:
Yes, I have asked the W3C to ensure that this URI resolves as it is actually specified in the Profiles Vocabulary [1] but there is an error with their URI hadling.

[1] https://www.w3.org/TR/dx-prof/#Role:schema